docs/src/introduction/getting-started.md

```diff
@@ -1 +1,131 @@
+- **Node IP Address:** Obtain the IP address of the running Miden-Node. This information can be acquired by contacting one of the Miden engineers.
+- **Miden-Client Installation:** You need to install the [Miden client](https://github.com/0xPolygonMiden/miden-client) and configure it to point to the remote node.
+
+## Step 1: Configuring the Miden-Client
+
+1. **Download the Miden-Client:** First, download the Miden client from its repository. Use the following command:
+
+   ```shell
+   git clone https://github.com/0xPolygonMiden/miden-client
+   ```
+
+2. **Navigate & Configure the Client:** Navigate to the client directory and modify the configuration file to point to the remote Miden-Node. You can find the configuration file at `./miden.toml`. In the `[RPC]` section replace the `endpoint = { host: }` field with the address provided by the Miden engineer.
+
+   ```shell
+   cd miden-client
+   ```
+
+   Configuration file example:
+
+   ```toml
+    [rpc]
+    endpoint = { protocol = "http", host = "<NODE_IP_ADDRESS>", port = 57291 }
+
+    [store]
+    database_filepath = "store.sqlite3"
+   ```
+
+3. **Build & install the Client:** install the client using cargo:
+
+   ```shell
+   cargo install --features testing --path .
+   ```
+
+   you should now be able to use the following command:
+
+   ```shell
+   miden-client --help
+   ```
+
+## Step 2: Setting-up the Miden-Client
+
+1. **Creating new accounts:** To be able to interact with the Miden node we will need to generate accounts. For this example we will be generating 2 accounts: A `basic-immutable` account and a `fungible-faucet`. You can generate new accounts using the following command:
+
+   ```shell
+   miden-client account new basic-immutable
+   miden-client account new fungible-faucet [...]
+   ```
+
+   Please refer to the documentation of the CLI
+
+2. **Listing accounts:** To view the newly created accounts we can run the following command:
+
+   ```shell
+   miden-client account -l
+   ```
+
+3. **Syncing node state:** The client needs to periodically query the node to receive updates about entities that might be important in order to run transactions. The way to do so is by running the `sync` command:
+
+   ```shell
+   miden-client sync-state -s
+   ```
+
+## Step 3: Minting an asset
+
+Since we have now synced our local view of the blockchain and have account information, we are ready to execute and submit tranasctions. For a first test, we are going to mint a fungible asset for a regular account.
+
+```shell
+miden-client tx new mint <regular-account-ID> <faucet-account-id> 1000
+```
+
+This will execute, prove and submit a transaction that mints assets to the node. The account that executes this transaction will be the faucet as was defined in the node's configuration file. In this case, it is minting `1000` fungible tokens to `<regular-account-ID-A>`.
```

Comment:
`the faucet as was defined in the node's configuration file.` Is the minting happening on the account that the client created above or with a faucet in the node's config file?